src/components/Map/StaticMap.scss

```diff
@@ -0,0 +1,50 @@
+@import "../variables";
+
+.static-map-wrapper {
+  display: flex;
+  // TODO: maybe remove this and let parent element decide the size
```

Comment:
Yea. Personally I like setting component sizes to 100% unless it's absolutely fixed size.

Reply:
After thinking about this a bit more, I think it makes sense to define a height for the map component. 

If the container of the map does not have a fixed size, then it is up to the map to determine its size. However, the map is a picture and it has constant width and height. This means that the map might be difficult to read on a smaller screen as the picture gets scaled down.

On the other hand, if the container has a fixed height, we could dynamically check the width of the container and use the API to get a new image that looks good on the current resolutions.